app/Http/Controllers/Base/AccountController.php

```diff
@@ -60,7 +65,7 @@ public function update(AccountDataFormRequest $request)
         } elseif ($request->input('do_action') === 'email') {
             $data['email'] = $request->input('new_email');
         } elseif ($request->input('do_action') === 'identity') {
-            $data = $request->only(['name_first', 'name_last', 'username']);
+            $data = $request->only(['name_first', 'name_last', 'username', 'language']);
```

Comment:
The language should be validated

Reply:
Yes, you're right. I'm on it.

Reply:
Nevermind, it's already being validated. Take a look at `gatherRules()` in `Models\User`